scripts/irods/controller.py

```diff
@@ -298,3 +304,12 @@ def delete_cache_files_by_name(*filepaths):
             os.unlink(path)
         except (IOError, OSError):
             l.warning(lib.indent('Error deleting cache file: %s'), path)
+
+def delete_s3_shmem():
+    # delete s3 shared memory if any exist 
+    s3_plugin_shmem = glob.glob(os.path.join(
+        paths.root_directory(),
+        'dev',
+        'shm',
+        '*s3-shm*'))
```

Comment:
we should consider namespacing these shmem files a bit more...

`*irods_s3-shm*` ?

Reply:
Good idea.  I will do that.

Reply:
Done

Reply:
You used `*irods-s3-shm*` rather than an underscore between `irods` and `s3`, like the others we use for the re_cache.

An underscore will keep them sorted/consistent.